core/node/da_clients/src/eigen/client.rs

```diff
@@ -203,4 +203,41 @@ mod tests {
         let retrieved_data = client.get_blob_data(&result.blob_id).await.unwrap();
         assert_eq!(retrieved_data.unwrap(), data);
     }
+
+    #[tokio::test]
+    async fn test_wait_for_finalization() {
+        let config = EigenConfig::Disperser(DisperserConfig {
+            custom_quorum_numbers: None,
+            disperser_rpc: "https://disperser-holesky.eigenda.xyz:443".to_string(),
+            eth_confirmation_depth: -1,
+            eigenda_eth_rpc: String::default(),
+            eigenda_svc_manager_address: "0xD4A7E1Bd8015057293f0D0A557088c286942e84b".to_string(),
+            blob_size_limit: 2 * 1024 * 1024, // 2MB
+            status_query_timeout: 1800,       // 30 minutes
```

Comment:
is the configuration documented somewhere? I found confusing that the numbers in status_query_timeout and status_query_interval (both related to the status_query) have different units.

Reply:
Done